client/my-sites/site-settings/date-time-format/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSite } from 'state/ui/selectors';
+
+export const DateTimeFormat = ( { translate, site } ) => {
+	const goBack = () => {
+		page( '/settings/general/' + site.slug );
+	};
+
+	return (
+		<div className="main main-column date-time-format" role="main">
+			<DocumentHead title={ translate( 'Manage Date and Time Format Configurations' ) } />
```

Comment:
Makes sense, but I still find it useful to audit every pr for now, to make sure everything  works as expected. Once we're more confident we'll probably activate on "needs review".